app.py

```diff
@@ -80,7 +80,7 @@ def start_convo(
             client=client,
             wip_reply=wip_reply,
             context=context,
-            user_id=context.actor_user_id,
+            user_id=context.user_id,
```

Comment:
Can you change this part?
```suggestion
            user_id=context.actor_user_id  or context.user_id,
```

Reply:
This still generates a runtime error when creating a new thread.
<img width="417" alt="actor_user_id" src="https://user-images.githubusercontent.com/62312786/225425109-39c9c318-d153-4a53-b7a2-0fa658baf067.png">

Reply:
According to https://stackoverflow.com/questions/51881324/does-python-have-a-shorthand-to-check-if-an-object-has-an-attribute, the `or` trick doesn't work with missing attributes.